backend/packages/Upgrade/.env.test

```diff
@@ -27,7 +27,7 @@ TYPEORM_CONNECTION=postgres
 TYPEORM_HOST=postgres
 TYPEORM_HOSTNAME_REPLICAS=[]
 TYPEORM_PORT=5432
-TYPEORM_USERNAME=postgres
+TYPEORM_USERNAME=doswalt
 TYPEORM_PASSWORD=postgres
 TYPEORM_DATABASE=postgres
 TYPEORM_SYNCHRONIZE=false
```

Comment:
Do you want to put your name here?

Reply:
ack no that wasn't supposed to be pushed